server/opts.go

```diff
@@ -227,8 +227,9 @@ type Options struct {
 	// Operating a trusted NATS server
 	TrustedKeys              []string              `json:"-"`
 	TrustedOperators         []*jwt.OperatorClaims `json:"-"`
-	AccountResolver          AccountResolver       `json:"-"`
-	AccountResolverTLSConfig *tls.Config           `json:"-"`
+	operatorJWT              []string
```

Comment:
I would move this one (and while at it `resolverPreloads`) down after the comment saying that here are the private fields.

Reply:
fixed